File: benchmark/cases/iops/iops.py
import os
import time


def iops(n: int) -> int:
    a = 0
    i = 1
    while i <= n:
        a = a + 1
        i = i + 1
    return a


t0 = time.perf_counter_ns()
ans = iops(int(os.environ["BENCH_SCALE"]))
t1 = time.perf_counter_ns()
print("iops a =", ans)
print("__bench_ns:", t1 - t0)

File: benchmark/cases/iops/iops.rs
use std::time::Instant;

fn iops(n: i64) -> i64 {
    let mut a = 0;
    let mut i = 1;
    while i <= n {
        a = a + 1;
        i = i + 1;
    }
    a
}

fn main() {
    let scale: i64 = std::env::var("BENCH_SCALE").unwrap().parse().unwrap();
    let t0 = Instant::now();
    let ans = iops(scale);
    let ns = t0.elapsed().as_nanos();
    println!("iops a = {}", ans);
    println!("__bench_ns: {}", ns);
}
